### ISSUE TITLE
Capturing The Turing Way's use/reuse/sharing/impact stories from our users, readers, contributors and collaborators

### ISSUE THREAD

**Issue by malvikasharan** [the-turing-way/the-turing-way#1659]
### Summary

In this issue, we are inviting comments from the community members a news item for the December newsletter.

This is also the place to add their _The Turing Way_ impact information in terms of its use, reuse, remix, spin-off projects, etc.

### What needs to be done?

- [ ] Share this issue on Slack to capture Newsletter items for December
- [ ] Capture response from Twitter, see [Kirstie's Tweet](https://twitter.com/kirstie_j/status/1326126705270984710?s=20)
- [ ] Share link in the newsletter and invite more impact stories

### Who can help?

* Anyone who may something to add

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

**Comment by rainsworth** [comment]
The Software Sustainability Institute’s Collaborations Workshop series brings together researchers, developers, innovators, managers, funders, publishers, policy makers, leaders and educators to explore best practices and the future of research software. Collaborations Workshop 2021 (CW21) will take place online from Tuesday, 30 March to Thursday, 1 April 2021 and registration is now open! The themes of the CW21 keynote presentations, discussion groups, mini-workshops, collaborative ideas, and hack day will be around the following areas of research software: FAIR Research Software, Diversity & Inclusion and Software Sustainability. To find out more information including registration, agenda, call for submissions and how to sponsor the unconference, please visit the CW21 website: http://bit.ly/ssi-cw21

**Comment by rainsworth** [comment]
The call for applications to the Software Sustainability Institute’s Fellowship Programme 2021 is now open! The Fellowship Programme provides funding for individuals who want to improve how research software is done in their domains and/or area of work. Being a Fellow also helps individuals develop skills and knowledge in their area around software sustainability, empowers them to speak about software sustainability issues and network with like-minded individuals from a wide variety of research areas. The Institute will be running a Fellows 2021 Launch Webinar on Thursday, 14 January 2021 to explain the programme in more detail and answer your questions. The deadline for applications is Friday, 5 February 2021. To find out more information including how to apply, schedule, example applications from successful Fellows, and words of encouragement, please visit the programme website: http://bit.ly/ssi-fellows2021

**Comment by 3mmaRand** [comment]
Yes, recommending to PGR students and integrated masters students 
https://3mmarand.github.io/BIO00058M-Data-science-2020/slides/00_intro_to_module.html#17

And basing much of https://cloud-span.york.ac.uk/  practice and handbook https://cloud-span.github.io/CloudSPAN-handbook/ on it

**Comment by malvikasharan** [comment]
Responses from Twitter:

Laura Ación
> I recommended and I know it was used in my group. I reference it in an upcoming review and also folks at @arphai_arg
 (part of @GlobalSAI4COVID) are using it to guide good open data science practice.

Ariel Rokem
> We cited TTW in a paper that came out this year

Jed Brown
> Deep linking students and colleagues.

Chris Holdgraf
> I have, happy to chat.

See the thread here: https://twitter.com/kirstie_j/status/1469060341606801410?s=20&t=A1VQoAdIHfv2m7wzgxUr9g

**Comment by malvikasharan** [comment]
Markus Löning
> We followed a lot of the recommendations when setting up our community structures in sktime (CoC, acknowledging contributions, etc) and mention it in our contributing guide as a useful resource for new contributors

Camila Rangel-Smith
> The Turing Way has been very important in designing the structure and syllabus of a data science module for the @lacongaphysics project. Also, when clases start next year is going to be part of the recommended literature.
> It has been a massive inspiration for the @TuringDStories
 project (which is still in it’s infancy but has big plans). We have borrowed so many ideas, from the general culture, to community building, contribution model, etc.

Laura Carter
> The @turingway code of conduct was one of the resources used by the volunteers who developed the code of conduct for @DataKindUK.

Ariel Rokem
> I'm currently writing a paper that uses this typology: https://the-turing-way.netlify.app/reproducible-research/overview/overview-definitions.html#table-of-definitions-for-reproducibility
Elizabeth DuPre
> I was pleasantly surprised to see the same terminology in this paper describing the NeurIPS reproducibility challenge ! https://arxiv.org/abs/2003.12206

Emma Robinson
> I just cited it in a grant proposal

Natalie Thurlby
> I used continuous integration for the first time at a book dash and I am a total convert: I use it on most projects now and it motiveates me to write more tests. I'm in a research support role, so I continue to spread the good word :).

**Comment by HeidiSeibold** [comment]
I have mentioned it in many of my talks:

- useR! keynote: https://youtu.be/dag9l0GFci8
- HDR UK talk: https://www.youtube.com/watch?v=hOgEd1KkA4A
- Others, e.g. https://youtu.be/T-OVRYKJ86k?t=1161

**Comment by aleesteele** [comment]
Restarting this issue to add:
- https://twitter.com/JamesDoesData/status/1537369814041231360
- https://twitter.com/alejo_coca/status/1527739283708297217

**Comment by malvikasharan** [comment]
@aleesteele please move this to #2017
